src/Client.php

```diff
@@ -65,6 +65,8 @@ public function query() {
     /**
      * Executes the given Query and returns the result
      *
+     * This method should not be called manually.
```

Comment:
Definitely, I just didn't know how to make the fancy reference in phpdoc and I thought this beat nothing. :)

I think we should "sync" all our API documentation at a later stage once things have settled.